### PR TITLE
NOISSUE - Make the Publish Configuration Access to Public

### PR DESCRIPTION
**Pull request by ianmuchyri**
Pull request title should be `JSSDK-XXX - description` or `NOISSUE - description` where XXX is ID of issue that this PR relate to.
Please review the [CONTRIBUTING.md](./CONTRIBUTING.md) file for detailed contributing guidelines.

### What does this do?
Makes the publishConfig access option in package.json set to public to state it is publishing to a public branch
### Which issue(s) does this PR fix/relate to?
Noissue
Put here `Resolves #XXX` to auto-close the issue that your PR fixes (if such)

### List any changes that modify/break current functionality
none
### Have you included tests for your changes?
no
### Did you document any new/modified functionality?
no

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@absmach/magistrala-sdk",
-  "version": "0.3.12",
+  "version": "0.3.14",
```
Comment (dborovcanin) [inline package.json:3]: @ianmuchyri Why did we skip `0.3.13`?

**Review by dborovcanin — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (ianmuchyri) [reply package.json:3]: I skipped it accidentally. But now since this `0.3.14` was already published, I don't know if we can be able go lower

Comment (dborovcanin) [reply package.json:3]: Let's unpublish 0.3.14.

Comment (dborovcanin) [reply package.json:3]: We may not be able to publish under `0.3.13` in a day or so because it's been unpublished recently.

Comment (ianmuchyri) [reply package.json:3]: Okay

Comment (ianmuchyri) [reply package.json:3]: I will revert to `0.3.12` then

Comment (ianmuchyri) [reply package.json]: I have reverted

**Pull request merged**